In [3]:
import pandas as pd

In [13]:
vl = pd.read_csv("validation_list.txt", header = None)

In [14]:
tl = pd.read_csv("testing_list.txt",header = None)

In [15]:
vl.shape

(6798, 1)

In [16]:
tl.shape

(6835, 1)

In [31]:
tl.shape[0] + vl.shape[0]

13633

In [17]:
tl

0
0      bed/0c40e715_nohash_0.wav
1      bed/0ea0e2f4_nohash_0.wav
2      bed/0ea0e2f4_nohash_1.wav
3      bed/105a0eea_nohash_0.wav
4      bed/1528225c_nohash_0.wav
...                          ...
6830  zero/fce96bac_nohash_1.wav
6831  zero/fe1916ba_nohash_0.wav
6832  zero/ffa76c4a_nohash_0.wav
6833  zero/ffb86d3c_nohash_0.wav
6834  zero/ffb86d3c_nohash_1.wav

[6835 rows x 1 columns]

In [25]:
used_commands = ['yes', 'no', 'up', 'down', 'left', 'right', 'on', 'off', 'stop', 'go']

In [27]:
tl.iloc[:, 0].map(lambda x: x.split("/")).map(lambda x: x[0] in used_commands).sum()

2567

In [28]:
vl.iloc[:, 0].map(lambda x: x.split("/")).map(lambda x: x[0] in used_commands).sum()

2577

In [29]:
n_obs = vl.iloc[:, 0].map(lambda x: x.split("/")).map(lambda x: x[0] in used_commands).sum() + tl.iloc[:, 0].map(lambda x: x.split("/")).map(lambda x: x[0] in used_commands).sum()

In [30]:
n_obs

5144

# Conclusions

There are 5144 audio files concerning selected 10 voice commands. It is already split in ~50-50 train-test split, but the training dataset will be bigger ( ~75\% of the labelled data).